lib/hal/video.c

```diff
@@ -230,6 +235,24 @@ BOOLEAN XVideoListModes(VIDEO_MODE *vm, int bpp, int refresh, void **p)
 		if(pVidMode->refresh != refresh)
 			continue;
 
+		if(dwAdapter == AV_PACK_HDTV)
+		{
+			bool is_hd = pVidMode->dwMode & 0x80000000;
+
+			/* If 480p support is enabled, reject 480i modes
+			   (forces a 480p mode to be picked) */
+			if(pVidMode->height == 480 && !is_hd && !allow_480i)
+				continue;
```

Comment:
There's no 480i mode in our video-mode table for HDTV adapter.

We need to adopt / add table from https://sourceforge.net/projects/openxdk/files/Patches%20and%20Drivers/Zagon_s%20480i%20patch/zagons-480i-patch.tar.gz/download in this or a separate PR first (or we potentially break applications)